source/quickstart.rst

```diff
@@ -0,0 +1,477 @@
+
+Quickstart
+**********
+
+The present tutorial is meant to provide a brief introduction to the use of PySCF to run a multitude of quantum chemical simulations. Starting with input parsing and uncorrelated Hartree-Fock theory for chemical Hamiltonians, we'll incrementally touch upon how to use the majority of methods and features offered by PySCF through a number of simple examples, which all make reference to specific use cases within the dedicated `examples <https://github.com/pyscf/pyscf/tree/master/examples>`_ directory. For brevity and so as to not repeat a number of function calls, please also note that the cells below often share objects in-between one another.
```

Comment:
Quickstart, not tutorial.

I would change the text to make it clear that the quickstart is aimed for people that already know what they want to do, e.g. people transferring from other codes.

Reply:
I get that, but I guess people will still use the section headers to get to what they want in the end? As in, no need to read stuff on mean-field calculations if one is only really interested in FCI? Anyways, I'll attempt to shorten this somewhat.